package.json

```diff
@@ -35,5 +35,8 @@
   },
   "devDependencies": {
     "dotenv-cli": "^6.0.0"
+  },
+  "engines": {
+    "node": "<17.0.0"
```

Comment:
Note for other reviewers is that this is here since Heimdall doesn't work on node v18 (already have an issue about this).  Heroku's buildpack uses 18 by default.  The version of the buildpack that uses 16 is being deprecated.  For awareness of others / to make the Heroku build work, we need to specify a compatible version of node.  This will be removed once we get it working on v18.

Reply:
https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version